CMake/IBMCompilers.cmake

```diff
@@ -54,3 +55,43 @@ IF(QMC_BITS MATCHES 64)
         "<CMAKE_AR> -X64 cr <TARGET> <LINK_FLAGS> <OBJECTS> " 
         "<CMAKE_RANLIB> <TARGET> ")
 ENDIF(QMC_BITS MATCHES 64)
+ENDIF(CMAKE_SYSTEM_NAME MATCHES "AIX")
+
+IF(CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64le")
+MESSAGE(STATUS "Power8+ system using xlC/xlc/xlf")
+
+ADD_DEFINITIONS( -Drestrict=__restrict__ )
+ADD_DEFINITIONS( -DADD_ )
+ADD_DEFINITIONS( -DINLINE_ALL=inline )
+
+# Clean up flags
+
+IF(CMAKE_C_FLAGS MATCHES "-qhalt=e")
+  SET(CMAKE_C_FLAGS "")
+ENDIF()
+IF(CMAKE_CXX_FLAGS MATCHES "-qhalt=e")
+  SET(CMAKE_CXX_FLAGS "")
+ENDIF()
+
```

Comment:
Yes, cmake populates those flags with "-qthreaded -qhalt=e" where -qthreaded is not doing anything and -qhalt=e is not supported anymore on the newer architecture. The other work-around would be to pass -DCMAKE_C_FLAGS="" and -DCMAKE_CXX_FLAGS="" to the cmake command which is not overly user-friendly.